instrumentation/github.com/aws/aws-lambda-go/otellambda/lambda.go

```diff
@@ -72,15 +72,15 @@ func (i *instrumentor) tracingBegin(ctx context.Context, eventJSON []byte) (cont
 	}
 	if lc != nil {
 		ctxRequestID := lc.AwsRequestID
-		attributes = append(attributes, semconv.FaaSExecution(ctxRequestID))
+		attributes = append(attributes, semconv.FaaSInvocationID(ctxRequestID))
 
 		// Some resource attrs added as span attrs because lambda
 		// resource detectors are created before a lambda
 		// invocation and therefore lack lambdacontext.
 		// Create these attrs upon first invocation
 		if len(i.resAttrs) == 0 {
 			ctxFunctionArn := lc.InvokedFunctionArn
-			attributes = append(attributes, semconv.FaaSID(ctxFunctionArn))
+			attributes = append(attributes, semconv.AWSLambdaInvokedARN(ctxFunctionArn))
```

Comment:
Is `faas.id` now `cloud.resource_id`?  https://github.com/open-telemetry/semantic-conventions/blob/4fcf4dfd795eccec8018f628c1331b3aa1aee99d/schemas/1.19.0#L14

Reply:
The [AWS spec](https://github.com/open-telemetry/semantic-conventions/blob/4fcf4dfd795eccec8018f628c1331b3aa1aee99d/docs/faas/aws-lambda.md#all-triggers) says this should have been `aws.lambda.invoked_arn` as far as I can tell.

Reply:
I think `faas.id` as incorrect from the start.

Reply:
cc @Aneurysm9

Reply:
I believe `aws.lambda.invoked_arn` is correct as a span attribute and because this is [using the invoked function ARN](https://github.com/open-telemetry/semantic-conventions/blob/4fcf4dfd795eccec8018f628c1331b3aa1aee99d/docs/resource/faas.md?plain=1#L62-L66) from the Lambda context.  `faas.id`/`cloud.resource_id` was likely correct in many cases, but could have been misleading in the case of a function with aliases.